src/main/java/org/example/service/JsonObjectService.java

```diff
@@ -0,0 +1,47 @@
+package org.example.service;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.example.model.Developer;
+
+import java.util.List;
+
+
+public class JsonObjectService {
+
+    private static DeveloperService developers = DeveloperService.getInstance();
+
+    private static JsonObjectService INSTANCE;
+
+
+    public String getJsonObject() {
```

Comment:
Take better title for this method.
(You return developers as an output)

The class already has `Json` in the title. You should not duplicate it in the methods titles.

Reply:
fixed